idaes/models/properties/modular_properties/base/tests/test_utility.py

```diff
@@ -559,3 +581,253 @@ def test_inherent_partial_pressure(self, frame2):
             get_concentration_term(frame2, "i1", log=True)
             is frame2.log_pressure_phase_comp
         )
+
+
+class TestIdentifyVLComponentList:
+    # Declare a base units dict to save code later
+    base_units = {
+        "time": pyunits.s,
+        "length": pyunits.m,
+        "mass": pyunits.kg,
+        "amount": pyunits.mol,
+        "temperature": pyunits.K,
+    }
+
+    # Dummy methods for properties
+    def set_metadata(self, b):
+        pass
+
+    def define_state(self, b):
+        pass
+
+    @declare_process_block_class("DummyParameterBlock")
+    class DummyParameterData(GenericParameterData):
+        def configure(self):
+            self.configured = True
+
+    @pytest.fixture()
+    def frame(self):
+        m = ConcreteModel()
+        m.params = DummyParameterBlock(
+            components={
+                "a": {},
```

Comment:
Are these supposed to be empty?

Reply:
Yes - this is a unit test. We need to define the components so the code runs but don't care about any actual properties.